docs/Security.md

```diff
@@ -38,6 +47,66 @@ users: [
     }
   }
 ]
+```
+
+## OAuth2 security module
+
+#### Configuration in following format:
+```
+authentication: {
+  method: "OAuth2"
+  clientSecret: ""
+  clientId: ""
+  authorizeUri: "https://github.com/login/oauth/authorize"
+  redirectUri: "http://localhost:3000"
+  accessTokenUri: "https://github.com/login/oauth/access_token"
+  profileUri: "https://api.github.com/user"
+  accessTokenParams: {
+    grant_type: "authorization_code"
+  }
+  headers: {
+    Accept: "application/json"
+  }
+  authorizeParams: {
+    response_type: "code"
+  }
+
+  usersFile: "./develConf/tests/oauth2-users.conf"
+}
+
+```
+
+#### Users file in following format:
+```
+users: [ //Special settings by user email
+  {
+    email: "some@email.com"
+    roles: ["Admin"]
+  }
+]
+
+rules: [
```

Comment:
You identify those "rules" by `roleName`, reference to this "rules" in users `roles` field. Why not to use everywhere "role" word?

Reply:
Changed - no we have fieldname role.